app/vminsert/netstorage/netstorage.go

```diff
@@ -463,6 +463,9 @@ type storageNode struct {
 	// The number of rows dropped on overload if -dropSamplesOnOverload is set.
 	rowsDroppedOnOverload *metrics.Counter
 
+	// The number of rows dropped on overload if -dropSamplesOlderThan is set.
```

Comment:
The comment is misleading - this is the number of dropped samples with too old timestamps outside the configured duration at `-dropSamplesOlderThan` command-line flag